Makefile

```diff
@@ -40,7 +40,17 @@ thrift-image:
 	docker pull $(THRIFT_IMG)
 	$(THRIFT) -version
 
-protocompile:
+protoc-image:
+	docker build -f proto/Dockerfile -t jaegertracing/jaeger-idl-protoc .
+
+protoc-gen-go-client: protoc-image
+	mkdir -p "${PWD}/go"
+	docker run -it --rm -v ${PWD}:/work jaegertracing/jaeger-idl-protoc
+	docker run -it --rm -v ${PWD}:/work jaegertracing/jaeger-idl-protoc proto/model.proto
+	docker run -it --rm -v ${PWD}:/work jaegertracing/jaeger-idl-protoc proto/model_test.proto
```

Comment:
I don't think we need to move model_test.proto here, this is a public IDL repository, while the test file is internal to jaeger repo.